src/app.rs

```diff
@@ -524,6 +524,22 @@ pub fn build_app() -> App<'static, 'static> {
                 ),
         );
 
+    if cfg!(unix) {
+        app = app.arg(
+            Arg::with_name("owner")
+                .long("owner")
+                .short("o")
+                .takes_value(true)
+                .value_name("user:group")
+                .help("Filter by owning user and/or group")
+                .long_help(
+                    "Filter files by their user and/or group. \
+                     Format: [(user|uid)][:(group|gid)]. Either side is optional. \
+                     Precede either side with a ! to exclude files instead.",
```

Comment:
Minor: `…either side with a '!' to exclude…`

Specifying the full format is great, but a few examples would make things even easier to understand, I think. Maybe:

```
Examples:
  --owner john
  --owner :students
  --owner '!john:students'
```
```